aten/src/ATen/native/transformers/hip/flash_attn/ck/mha_fwd_ck.hip

```diff
@@ -0,0 +1,359 @@
+/******************************************************************************
+ * Copyright (c) 2024, Tri Dao.
+ ******************************************************************************/
+
+#include <ATen/native/transformers/hip/flash_attn/flash_common_hip.hpp>
+#include <third_party/composable_kernel/example/ck_tile/01_fmha/fmha_fwd.hpp>
```

Comment:
When including ck headers, can we only use the path inside composable. Or at least get rid of the third_party?